src/units/unit.cpp

```diff
@@ -434,12 +434,51 @@ void unit::init(const config& cfg, bool use_traits, const vconfig* vcfg)
 		for(const vconfig& e : events) {
 			events_.add_child("event", e.get_config());
 		}
+		const vconfig::child_list& abilities_tags = vcfg->get_children("abilities");
+		for(const vconfig& abilities_tag : abilities_tags) {
+            for(const auto& [key, child] : abilities_tag.all_ordered()) {
```

Comment:
Indentation. Also on line 450.

Reply:
I wonder if there is a good story behind how we ended up with such an interesting mix of indentation in this project; with schema configs and C++ using tabs while Lua, Python and WML use four spaces for indentation. :)

Reply:
It is not that simple even. GUI WML is back to tabs, and some of Lua uses spaces while normally Lua uses tabs.

Reply:
We even have a tool to notice that files are using both tabs and spaces for indentation, but there's too many affected C++ files to enable it for C++. So data/tools/check_mixed_indent only gets run on Lua files, and the schema stuff. :(

Reply:
I think the short answer is: nobody can agree on indentation styles.